cmake-modules/AddCmockaFunctions.cmake

```diff
@@ -0,0 +1,98 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Why was this cmake module file added/checked-in as part of the bug fix? Please make sure to keep your PRs specific to the issue being resolved.

Reply:
shouldn't be part of this PR.  I will remove it

Reply:
https://github.com/Azure/azure-sdk-for-c/pull/1375

Reply:
Reverted in https://github.com/Azure/azure-sdk-for-c/pull/1377